Assets/Scripts/PlayerController.cs

```diff
@@ -53,6 +53,7 @@ void Update()
             else
             {
                 moveDir.y = 0.0f;
+                animator.SetBool("Jump_b", false);
```

Comment:
This fixes a weird bug with perpetual player jumping when pressing spacebar and letting go quickly

Reply:
if it works it works lol